src/Logging/messages.lua

```diff
@@ -9,7 +9,10 @@ return {
 	cannotConnectChange = "The %s class doesn't have a property called '%s'.",
 	cannotConnectEvent = "The %s class doesn't have an event called '%s'.",
 	cannotCreateClass = "Can't create a new instance of class '%s'.",
+	cannotYield = "Attempted to yield in a synchronous thread.",
 	computedCallbackError = "Computed callback error: ERROR_MESSAGE",
+	computedCannotYield = "Cannot yield inside a Computed callback function.",
+	computedPairsCannotYield = "Cannot yield inside a ComputedPairs processor function.",
```

Comment:
For consistency, it's probably better to rename `computedPairsCannotYield` to `pairsProcessorCannotYield`. This also ties in with the destructor also being prevented from yielding.

Reply:
Working on this, ended up encountering some nasty bugs when calling the destructor after writing some test cases for it, which I'm working on.